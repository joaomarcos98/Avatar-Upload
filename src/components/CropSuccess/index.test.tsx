import { CropSuccess } from "."
import { renderWithTheme } from "../../utils/test/renderWithTheme"

describe("Crop success component", () => {
    it("should be renders all elements correctly", () => {
        const { container } = renderWithTheme(
            <CropSuccess
                onAccept={jest.fn()}
                onReject={jest.fn()}
                zoom="20"
                imageUrl="any-image"
            />
        )
        expect(container.querySelector(".dropzone")).toBeInTheDocument()
        expect(container.querySelector(".crop-image")).toBeInTheDocument()
        expect(container.firstChild).toMatchSnapshot()
    })
})
