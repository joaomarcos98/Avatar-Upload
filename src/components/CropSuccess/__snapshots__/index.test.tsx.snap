// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Crop success component should be renders all elements correctly 1`] = `
.c1 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 113px;
  height: 113px;
  border-radius: 72px;
  overflow: hidden;
  background-color: #FFFFFF;
  z-index: 1;
}

.c2 {
  width: 200px;
  height: 200px;
  -webkit-transform: scale(calc(20 / 20));
  -ms-transform: scale(calc(20 / 20));
  transform: scale(calc(20 / 20));
}

.c3 {
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding-left: 50px;
}

.c4 {
  font-size: 16px;
  color: #495567;
  line-height: 25px;
  margin: 0;
}

.c5 {
  min-width: 16px;
  min-height: 12px;
  padding-right: 12px;
}

.c6 {
  color: #677489;
  line-height: 25px;
  font-size: 14px;
  margin: 0;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  padding: 32px;
  position: relative;
}

<div
  class="c0 crop-success"
>
  <div
    class="c1 crop-image"
  >
    <img
      alt="Avatar image"
      class="c2"
      src="any-image"
    />
  </div>
  <div
    class="c3 dropzone"
    tabindex="0"
  >
    <input
      accept="image/*"
      autocomplete="off"
      id="dropzone"
      style="display: none;"
      tabindex="-1"
      type="file"
    />
    <h3
      class="c4"
    >
      <img
        alt="Image icon"
        class="c5"
      />
       Organization Logo
    </h3>
    <p
      class="c6"
    >
      Drop the image here or click to browse.
    </p>
  </div>
</div>
`;
